doc/build-freebsd.md

```diff
@@ -71,10 +71,12 @@ pkg install qt5-buildtools qt5-core qt5-gui qt5-linguisttools qt5-testlib qt5-wi
 
 ###### libqrencode
 
-The GUI can encode addresses in a QR Code. To build in QR support for the GUI, install `libqrencode`. Skip if not using the GUI or don't want QR code functionality.
+The GUI can encode addresses in a QR code. To build in QR code support for the GUI, install libqrencode:
```

Comment:
I think this should state the GUI will not compile without this dependency unless it is explicitly disabled.

Reply:
I've tried to maintain a positive tone and used the phrase: "Skip this step if not using the GUI or do not want QR code functionality. In the latter case, pass `-DWITH_QRENCODE=OFF`." :)

Would you mind suggesting the exact wording you think suits better?

Reply:
Bitcoin Core includes a GUI built with the cross-platform Qt Framework. To compile the GUI, we need to install the necessary parts of Qt **and 'libqrencode'**. Skip if you don't intend to use the GUI.
```bash
pkg install qt5-buildtools qt5-core qt5-gui qt5-linguisttools qt5-testlib qt5-widgets
```

#### libqrencode

The GUI will be able to encode addresses in QR codes unless this feature is explicitly disabled. To install `libqrencode`, run:
```bash
pkg install libqrencode
```
Otherwise, If you don't need QR encoding support, use the `-DWITH_QRENCODE=OFF` option to disable this feature in order to compile the GUI.

Reply:
Thanks! Reworked using your suggestion.